### PR TITLE
fixed:strict typescript errors

### PR DESCRIPTION
**Pull request by abhishekkumar08**
- [x] Card
- [x] Checkbox
 
Refr: #2929

### CODE REVIEW

File: src/card/CardImage.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const CardImage: RneFunctionComponent<ImageProps> = ({ style, ...props }) => {
 
 const styles = StyleSheet.create({
   image: {
-    width: null,
+    width: undefined,
```
Comment (arpitBhalla) [inline src/card/CardImage.tsx:13]: it must be null

Comment (abhishekkumar08) [reply src/card/CardImage.tsx:13]: okay, then finding a better sloution for this.

Comment (arpitBhalla) [reply src/card/CardImage.tsx:13]: Muzhe bhi batana😅

Comment (flyingcircle) [inline src/card/CardImage.tsx:13]: This is changing the behavior of the component. Either we should turn on ignoring null checks or find a different solution.

**Review by flyingcircle — CHANGES_REQUESTED** [review]

File: src/card/CardImage.tsx (hunk 1 above)
Comment (abhishekkumar08) [reply src/card/CardImage.tsx]: Okay would gonna do that!

**Comment by abhishekkumar08** [comment]
@flyingcircle Please review. Thanks

**Pull request merged**